src/pages/inquiry/request/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { InquiryApi } from '@/api'
+import { Button, LineInput } from '@/components/ui'
+import BaseLayout from '@/layouts/BaseLayout'
+import { NextPage } from 'next'
+import Head from 'next/head'
+import { useEffect, useRef, useState } from 'react'
+import './RequestPage.scss'
+
+const InquiryRequestPage: NextPage<void> = () => {
+  const [title, setTitle] = useState('')
+  const [email, setEmail] = useState('')
+  const [body, setBody] = useState('')
+  const [isSubmitted, setIsSbumitted] = useState(false)
+
+  const titleRef = useRef<HTMLInputElement>(null)
+  const emailRef = useRef<HTMLInputElement>(null)
+  const bodyRef = useRef<HTMLTextAreaElement>(null)
+
+  function focusTitle(): void {
+    titleRef.current.focus()
+  }
+
+  function focusEmail(): void {
+    emailRef.current.focus()
+  }
+
+  function focusBody(): void {
+    bodyRef.current.focus()
+  }
+  async function submit(): Promise<void> {
+    if (isSubmitted) {
+      return
+    }
+    setIsSbumitted(true)
+    const result = await InquiryApi.post(title, body, email)
+    if (result) {
+      window.location.href = '/inquiry'
+      alert('Submitted')
+    } else {
+      setIsSbumitted(false)
+      alert('Error')
+    }
```

Comment:
You need to handle errors more specifically.
An email format error, for instance.